### PR TITLE
options: allow to opt out of '--cfg fuzzing'

### LINKED ISSUE #273: Allow to disable `--cfg fuzzing`
One of the dependencies of my project would deactivate some features on `#[cfg(fuzzing)]` which i'd like to keep.
Is there a way to opt out of `--cfg fuzzing`? If not would it be possible to add one?

### PR DESCRIPTION
**Pull request by darosior**
Fixes #273

### CODE REVIEW

File: src/options.rs [hunk 1]
```diff
@@ -128,6 +128,11 @@ pub struct BuildOptions {
     /// optimized targets. This flag allows you to opt out of it.
     #[structopt(long)]
     pub strip_dead_code: bool,
+
+    /// By default a 'fuzzing' compilation configuration is set. This flag
+    /// allows you to opt out of it.
+    #[structopt(long)]
+    pub unset_cfg: bool,
```
Comment (Manishearth) [inline src/options.rs:135]: no-cfg-fuzzing maybe?

Comment (darosior) [reply src/options.rs]: Better, done. Thanks!

**Review by Manishearth — APPROVED** [review]

**Comment by Manishearth** [comment]
I'll wait for @fitzgen to have a look as well and let him merge

File: src/options.rs [hunk 2]
```diff
@@ -128,6 +128,11 @@ pub struct BuildOptions {
     /// optimized targets. This flag allows you to opt out of it.
     #[structopt(long)]
     pub strip_dead_code: bool,
+
+    /// By default a 'fuzzing' compilation configuration is set. This flag
+    /// allows you to opt out of it.
```
Comment (fitzgen) [inline src/options.rs:133]: I think users will be able to better recognize what we are talking about here if we have the `cfg(..)` wrapper, like so:

```suggestion
    /// By default, the `cfg(fuzzing)` compilation configuration is set. This flag
    /// allows you to opt out of it.
```

**Review by fitzgen — APPROVED** [review]
LGTM, thanks! Just one tiny suggestion below

File: src/options.rs (hunk 2 above)
Comment (darosior) [reply src/options.rs]: Applied :)

**Review by Manishearth — APPROVED** [review]

**Comment by fitzgen** [comment]
Thanks!

**Pull request merged**